ui/src/main/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodPresenter.java

```diff
@@ -110,24 +109,24 @@ public void loadCurrency(final boolean forceUpdate, @NonNull final String countr
         }
 
         mUserRepository.loadUser(new UserRepository.LoadUserCallback() {
-            @Override
             public void onUserLoaded(@NonNull final HyperwalletUser user) {
                 mTransferMethodConfigurationRepository.getKeys(
                         new TransferMethodConfigurationRepository.LoadKeysCallback() {
                             @Override
-                            public void onKeysLoaded(
-                                    @Nullable final HyperwalletTransferMethodConfigurationKeyResult result) {
+                            public void onKeysLoaded(@Nullable final HyperwalletTransferMethodConfigurationKey key) {
                                 if (!mView.isActive()) {
                                     return;
                                 }
-                                List<String> transferMethodCurrencies = result.getCurrencies(countryCode);
-                                List<String> transferMethodTypes = result.getTransferMethods(
-                                        countryCode, transferMethodCurrencies.get(0), user.getProfileType());
+                                List<com.hyperwallet.android.model.meta.keyed.Currency> transferMethodCurrencies =
+                                        new ArrayList<>(key.getCurrencies(countryCode));
```

Comment:
is it possible `key.getCurrencies(countryCode)` returning null? If so we could have a npe here